DeskKitSwiftExample/DeskKitSwiftExample/SplitViewController.swift

```diff
@@ -0,0 +1,263 @@
+    
+    private func newTopicsViewController() -> DKTopicsViewController {
+        let controller = DKSession.newTopicsViewController()
+        
+        controller.delegate = self
+        controller.title = NSLocalizedString("Topics", comment: "Topics")
+        controller.toolbarItems = contactUsToolbarItems
+        
+        return controller
+    }
+    
+    private func newArticlesViewController() -> DKArticlesViewController {
+        let controller = DKSession.newArticlesViewController()
+        controller.toolbarItems = contactUsToolbarItems
+        return controller
+    }
+    
+    private func newArticleDetailViewController() -> DKArticleDetailViewController {
+        let controller = DKSession.newArticleDetailViewController()
+        controller.toolbarItems = contactUsToolbarItems
+        return controller
+    }
+    
+    private var contactUsToolbarItems: [UIBarButtonItem] {
+        let spacer1 = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
+        let spacer2 = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
+        let contactUsButton = UIBarButtonItem(title: NSLocalizedString("Contact Us", comment: "Contact Us"), style: .plain, target: self, action: #selector(contactUsButtonTapped(sender:)))
+        contactUsButtonIndex = 1
+        
+        return [spacer1, contactUsButton, spacer2]
+    }
+    
+    private func newEmptyViewController() -> UIViewController {
+        return UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "DKEmptyViewController")
+    }
+    
+    // MARK: Action Sheet
+    
+    func contactUsButtonTapped(sender: NSObjectProtocol) {
+        openActionSheet()
+    }
+    
+    private func openActionSheet() {
+        guard let toolbarItems = masterNavigationController?.topViewController?.toolbarItems where contactUsButtonIndex < toolbarItems.count else { return }
+        
+        let contactUsSheet = DKSession.newContactUsAlertController(callHandler: { (callAction) in
+            guard let phoneURL = DKSession.sharedInstance().contactUsPhoneNumberURL else { return }
+            
+            UIApplication.shared().openURL(phoneURL)
+            }) { (emailAction) in
+                self.alertControllerDidTapEmailUs()
+        }
+        
+        let contactUsButton = toolbarItems[contactUsButtonIndex]
+        contactUsSheet.popoverPresentationController?.barButtonItem = contactUsButton
+        present(contactUsSheet, animated: true, completion: nil)
+    }
+    
+    private func alertControllerDidTapEmailUs() {
+        let contactUsVC = DKSession.sharedInstance().newContactUsViewController()
+        contactUsVC.delegate = self
+        
+        /*
+        // Example of adding custom fields
+        // 1. Grab initial custom fields populated from DeskKitSettings.plist
+        let customFields = NSMutableDictionary(dictionary:contactUsVC.customFields)
```

Comment:
would be great to do it in pure swift way, something like:
```
var customFields = contactUsVC.customFields as [NSObject: AnyObject]
customFields += dynamicCustomFields
```
Unfortunately, Seems like it's not possible